src/main/java/sirius/biz/analytics/flags/jdbc/BitwiseConstraint.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.analytics.flags.jdbc;
+
+import sirius.biz.analytics.flags.PerformanceFlag;
+import sirius.biz.analytics.flags.PerformanceFlagged;
+import sirius.db.jdbc.SmartQuery;
+import sirius.db.jdbc.constraints.SQLConstraint;
+
+/**
+ * Provides a constraint which performs a bitwise check in order to determine if a performance flag is toggled.
+ */
+class BitwiseConstraint extends SQLConstraint {
```

Comment:
this sounds pretty generic and has performance flag specific implementation. maybe a more specific name is possible? Or a more generic implementation in sirius-db?